### PR TITLE
[GiDOutput] Cleanup SingleFile and Ascii

### PR DESCRIPTION
**Pull request by miguelmaso**
**📝 Description**
Small leftover. This PR removes an extra `_0` in the file name when it is `SingleFile` and `GiD_PostAscii`.

**🆕 Changelog**
- Remove extra `_0`

### CODE REVIEW

**Comment by miguelmaso** [comment]
@KratosMultiphysics/structural-mechanics @KratosMultiphysics/geomechanics sorry for the noise, this is almost an OCD with gid and clear filenames. Just removing an extra `_0`

@roigcarlo I've modified the `.gitignore`

@loumalouomega I've added an `auxiliar`, sorry for the typo

File: .gitignore [hunk 1]
```diff
@@ -79,3 +79,7 @@ bin/*
 
 # External downloaded libs (TPL downloaded libs, etc...)
 external_libraries/tetgen/*
+
+# Inclusion of post files for testing
+!**/auxiliar_files_for_python_unittest/**/*.post.res
```
Comment (philbucher) [inline .gitignore]: This should not be here, the files should be removed by the test

**Review by philbucher — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (miguelmaso) [reply .gitignore]: > This should not be here, the files should be removed by the test

Then, this [file](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/auxiliar_files_for_python_unittest/gid_io/ref_gid_io_gp_active_only_0.post.msh) and some others must be deleted

Let me check what's happening with the test

**Comment by loumalouomega** [comment]
> @loumalouomega I've added an `auxiliar`, sorry for the typo

Nooooooooooooo

File: .gitignore (hunk 1 above)
Comment (miguelmaso) [reply .gitignore]: Fixed. Renamed:

- `.post.res` to `.post.res.ref`
- `.post.msh` to `.post.msh.ref`

**Review by philbucher — APPROVED** [review]

**Comment by miguelmaso** [comment]
> > @loumalouomega I've added an `auxiliar`, sorry for the typo
> 
> Nooooooooooooo

removed, thanks to @philbucher review

**Pull request merged**